Altis_Life.Altis/core/functions/fn_handleDamage.sqf

```diff
@@ -20,31 +20,28 @@ if (!isNull _source) then {
     if (_source != _unit) then {
         if (currentWeapon _source in ["hgun_P07_snds_F","arifle_SDAR_F"] && _projectile in ["B_9x21_Ball","B_556x45_dual"]) then {
             if (side _source isEqualTo west && playerSide isEqualTo civilian) then {
-                _damage = 0;
-                if (alive player && !life_istazed && !life_isknocked && !(_unit getVariable ["restrained",false])) then {
-                    private ["_distance"];
-                    _distance = 35;
-                    if (_projectile == "B_556x45_dual") then {_distance = 100;};
-                    if (_unit distance _source < _distance) then {
-                        if !(isNull objectParent player) then {
-                            if (typeOf (vehicle player) == "B_Quadbike_01_F") then {
-                                player action ["Eject",vehicle player];
+                _damage = (damage player);
+                if (playerSide isEqualTo civilian) then {
```

Comment:
Should line 24 be the following because of the upcoming OPFOR side? Ping: @BoGuu 

```sqf
                if (playerSide isEqualTo civilian || {playerSide isEqualTo east}) then {
```